### PR TITLE
Add workflow to suggest skipping tests

### PR DESCRIPTION
**Pull request by senekor**

### CODE REVIEW

File: .github/workflows/no-important-files-changed.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          done
+
+          echo "important_files_changed=false" >> "$GITHUB_OUTPUT"
+
+      - name: Suggest to add [no important files changed]
+        if: steps.check.outputs.important_files_changed == 'true'
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
+        with:
+          script: |
+            const body = "This PR touches files which probably affect the outcome of the tests of an exercise. If this is not the case, please add the following to the merge-commit message. Copy-paste to avoid typos. This will prevent tests from rerunning unnecessarily. For more information, refer to the [documentation](https://exercism.org/docs/building/tracks#h-avoiding-triggering-unnecessary-test-runs).\n```\n[no important files changed]\n```"
```
Comment (ErikSchierboom) [inline .github/workflows/no-important-files-changed.yml:62]: We probably want to tweak the wording a bit, but that's for later.

**Review by ErikSchierboom — COMMENTED** [review]
Great!

File: .github/workflows/no-important-files-changed.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Check if important files changed
+        id: check
+        env:
+          TARGET_BRANCH: ${{ github.base_ref }}
+        run: |
+          set -exo pipefail
```
[the remark this answers is not in the document]
Comment (senekor) [reply .github/workflows/no-important-files-changed.yml]: done :+1:

File: .github/workflows/no-important-files-changed.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+name: No important files changed
+
+on:
+  pull_request:
+    types: [opened]
```
[the remark this answers is not in the document]
Comment (senekor) [reply .github/workflows/no-important-files-changed.yml:5]: I think the idea was to only run the script when the PR is created because there is (currently) no deduplication logic. If it runs again on every PR change, I would expect multiple messages to be generated on a single PR (annoying). And I thought it's unlikely that a PR is created that doesn't touch the tests at all, but then later changes to tests are added. Not that it's impossible.

I think if we want to cover that situation as well, we need to make sure the message is only generated once.

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply .github/workflows/no-important-files-changed.yml:5]: Ah you're right about the multiple messages, that's annoying. Let's keep it like this then. BTW, `edited` does not main it runs on every PR change, that's a separate push event.

File: .github/workflows/no-important-files-changed.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+          done
+
+          echo "important_files_changed=false" >> "$GITHUB_OUTPUT"
+
+      - name: Suggest to add [no important files changed]
+        if: steps.check.outputs.important_files_changed == 'true'
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
+        with:
+          script: |
+            const body = "This PR touches files which probably affect the outcome of the tests of an exercise. If this is not the case, please add the following to the merge-commit message. Copy-paste to avoid typos. This will prevent tests from rerunning unnecessarily. For more information, refer to the [documentation](https://exercism.org/docs/building/tracks#h-avoiding-triggering-unnecessary-test-runs).\n```\n[no important files changed]\n```"
```
Comment (iHiD) [inline .github/workflows/no-important-files-changed.yml:61]:
```suggestion
            const body = "This PR touches files which potentially affect the outcome of the tests of an exercise. This will cause all students' solutions to affected exercises to be re-tested.\n\nIf this PR does **not** affect the result of the test (or, for example, adds an edge case that is not worth rerunning all tests for), **please add the following to the merge-commit message** which will stops student's tests from re-running. Please copy-paste to avoid typos.\n```\n[no important files changed]\n```\n\n For more information, refer to the [documentation](https://exercism.org/docs/building/tracks#h-avoiding-triggering-unnecessary-test-runs). If you are unsure whether to add the message or not, please ping `@exercism/maintainers-admin` in a comment. Thank you!"
```
